examples/cropped_text.rs

```diff
@@ -0,0 +1,59 @@
+use embedded_canvas::Canvas;
+use embedded_graphics::{
+    pixelcolor::{Rgb555},
+    prelude::*,
+    primitives::{PrimitiveStyle, Rectangle},
+};
+use embedded_graphics_simulator::{
+    BinaryColorTheme, OutputSettingsBuilder, SimulatorDisplay, Window,
+};
+
+pub const DISPLAY_240P: Size = Size::new(320, 240);
+pub const DISPLAY_360P: Size = Size::new(480, 360);
+pub const DISPLAY_720P: Size = Size::new(1280, 720);
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    /* Algorithms:
```

Comment:
instead of `Algorithms` you can write e.g.
We draw:
1. A box with an outline
2. ....
3. ....

Reply:
Thanks, Lacho.
I have corrected it.